core/cap-0038.md

```diff
@@ -810,23 +810,21 @@ if !exists(tlPool)
 lp = loadLiquidityPool(lpwo.liquidityPoolID)
 cp = lp.constantProduct()
 
-tlA = loadTrustLine(sourceAccount, cp.assetA)
-tlB = loadTrustLine(sourceAccount, cp.assetB)
-if !exists(tlA) || !exists(tlB)
-    Fail with LIQUIDITY_POOL_WITHDRAW_NO_TRUST
-
 reserveA = cp.reserveA
 reserveB = cp.reserveB
 total = cp.totalPoolShares
 
-amount = cp.amount
+amount = lpwo.amount
```

Comment:
I can't comment on lines 835,836 but I think those should also use `amount`.

Reply:
fixed